engine/language-server/src/main/scala/org/enso/languageserver/runtime/VisualizationConfiguration.scala

```diff
@@ -99,11 +103,15 @@ object VisualizationConfiguration {
             expression <- cursor
               .downField(CodecField.Expression)
               .as[MethodPointer]
+            visualizationModule <- cursor
+              .downField(CodecField.VisualizationModule)
+              .as[String]
```

Comment:
You made the `visualizationModule` field required. Please update the `protocol-language-server.md` API doc.

Reply:
Documentation updated in 7b271889afe